tools/serverpod_cli/lib/src/generator/serverpod_code_generator.dart

```diff
@@ -87,6 +88,32 @@ abstract class ServerpodCodeGenerator {
     return allFiles.keys.toList();
   }
 
+  /// Generate openapi Schema from [OpenApiGenerator]
+  /// Return a generated file
+  static Future<String> generateOpenApiSchema({
+    required ProtocolDefinition protocolDefinition,
+    required GeneratorConfig config,
+    required CodeGenerationCollector collector,
+  }) async {
+    collector.generatedFiles.clear();
+    OpenApiGenerator generator = const OpenApiGenerator();
+    Map<String, String> file = generator.generateOpenApiSchema(
+        protocolDefinition: protocolDefinition, config: config);
+
+    try {
+      log.debug('Generating ${file.keys}');
```

Comment:
Even though its only one file in the `Map` I would write this as a loop. Another option is to use another return type as using the `Map<String, String>` would indicate that it is ok to return multiple files. If someone changes the generate method in the future, this would break things in a way that may not be immediately clear to the developer.